python/ray/experimental/state.py

```diff
@@ -558,19 +573,35 @@ def micros_rel(ts):
             delta_info["function_name"] = info["function_name"]
             delta_info["worker_id"] = info["worker_id"]
             worker = workers[info["worker_id"]]
+            task_t_info = task_table[task_id]
+            task_spec = task_table[task_id]["TaskSpec"]
+            task_spec["Args"] = [oid.hex() if type(oid) is not int else oid
+                                 for oid in task_t_info["TaskSpec"]["Args"]]
+            task_spec["ReturnObjectIDs"] = [oid.hex() for oid in
+                                            (task_t_info["TaskSpec"]
+                                             ["ReturnObjectIDs"])]
+            task_spec["LocalSchedulerID"] = task_t_info["LocalSchedulerID"]
+
+            total_info = dict(delta_info, **task_spec)
+
+            parent_info = task_info.get(
+                            task_table[task_id]["TaskSpec"]["ParentTaskID"])
+            worker = workers[info["worker_id"]]
+
             if breakdowns:
                 if "get_arguments_end" in info:
                     get_args_trace = {
                         "cat": "get_arguments",
                         "pid": "Node " + str(worker["node_ip_address"]),
                         "tid": info["worker_id"],
-                        "id": str(worker),
+                        "id": str(task_id),
                         "ts": micros_rel(info["get_arguments_start"]),
                         "ph": "X",
                         "name": info["function_name"] + ":get_arguments",
-                        "args": delta_info,
+                        "args": total_info,
                         "dur": micros(info["get_arguments_end"] -
-                                      info["get_arguments_start"])
+                                      info["get_arguments_start"]),
+                        "cname": "rail_idle"
```

Comment:
Is there a description somewhere of what the json format should be and what all of these fields mean? If so, can we include a link in the docstring for this function to the explanation?